apps/base-docs/base-learn/docs/hardhat-verify/hardhat-verify-sbs.md

```diff
@@ -42,7 +42,7 @@ In order to obtain an Etherscan API key, visit [Etherscan](https://etherscan.io/
 
 Then, go to [https://etherscan.io/myapikey](https://etherscan.io/myapikey) and create an API key by clicking the **Add** button:
 
-![Add key](../../assets/images/hardhat-verify/harhat-verify-create-key.png)
```

Comment:
the filename is correct (the typo is also in the filename), so please either revert or also change the name of the file